react/ConditionLayoutProduct.tsx

```diff
@@ -70,6 +72,9 @@ export const HANDLERS: Handlers<SubjectValues, SubjectArgs> = {
 
     return specification.values.includes(String(args?.value))
   },
+  isProductAvailable({ values }) {
+    return values.isProductAvailable
+  },
```

Comment:
And the availability logic should be here, not in the layout render method, so it's executed only when needed.